bundle/manifests/update-service-operator.clusterserviceversion.yaml

```diff
@@ -77,6 +77,98 @@ spec:
   install:
     spec:
       clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          - services
+          - secrets
```

Comment:
I cannot see where we list/watch `secrets` in the code. But it [errored](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_cincinnati-operator/191/pull-ci-openshift-cincinnati-operator-master-operator-e2e-hypershift-local-graph-data/1808793702563319808/artifacts/operator-e2e-hypershift-local-graph-data/e2e-test/artifacts/inspect/namespaces/openshift-updateservice/pods/updateservice-operator-5999758c8c-6hg24/updateservice-operator/updateservice-operator/logs/current.log) out without it.

```text
2024-07-04T09:51:16.153687092Z W0704 09:51:16.153638       1 reflector.go:424] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:262: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:openshift-updateservice:updateservice-operator" cannot list resource "secrets" in API group "" in the namespace "openshift-config"
```